docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,7 +1,198 @@
 ---
-title: "Getting started with Dagster Cloud | Dagster Docs"
+title: Getting started with Dagster Cloud | Dagster Cloud
 
-noindex: True
+platform_type: "cloud"
 ---
 
-# Getting started with Dagster Cloud
+# Getting Started with Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Welcome to Dagster Cloud!
+
+In this guide, we'll give you everything you need to get up and running with Dagster Cloud. We'll walk you through setting up your Dagster Cloud account, configuring deployment, loading your code, launching your first run, and configuring continuous integration.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need access to your Dagster code.
```

Comment:
we say "your dagster code" here as though you need to have it ready to go, but then the next line says we're happy to provide an example - this could also be something like "you'll need some Dagster code to run" or "You'll be executing some Dagster code during this guide"